src/content/guides/getting-started.md

```diff
@@ -32,6 +32,14 @@ W> The minimum supported Node.js version to run webpack 5 is 10.13.0 (LTS)
 
 ## Basic Setup
 
+<aside class="preview">
+  <h6 class="preview__prefix">live preview</h6>
+  <p>Preview this example live on <a href="https://stackblitz.com">StackBlitz</a>.</p>
+  <a href="https://stackblitz.com/github/d3lm/webpack.js.org/tree/feat/add-stackblitz-live-previews/examples/getting-started/basic-setup?runScript=start" target="_blank">
+    <img src="https://developer.stackblitz.com/img/open_in_stackblitz.svg">
+  </a>
+</aside>
```

Comment:
I think this block is too early here.

I think these buttons should represent the source code state that you would have at this position of the guide.

So this example should be at the end of the chapter.

Reply:
Updated.